packages/react-make-styles/README.md

```diff
@@ -129,6 +129,29 @@ const useStyles = makeStyles({
 });
 ```
 
+### 🎞 `keyframes` (animations)
+
+`keyframes` are supported via `animationName` property that can be defined as an object or an array of objects:
+
+```tsx
+import { makeStyles } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    animationIterationCount: 'infinite',
```

Comment:
nit: add an example for the array of objects

Reply:
Good call, added in e2f93e4.

---

I also discovered another problem when I was creating an example, we used `" "` (space) as a delimiter, but we should use comma (48f14a5).

https://developer.mozilla.org/en-US/docs/Web/CSS/animation-name